rmw_opensplice_cpp/src/functions.cpp

```diff
@@ -66,11 +68,10 @@ rmw_init()
 {
   DDS::DomainParticipantFactory_var dp_factory = \
     DDS::DomainParticipantFactory::get_instance();
-  if (!dp_factory)
-  {
+  if (!dp_factory) {
```

Comment:
The conclusion from the last discussion was that they are on the next line for things like:
* namespace
* class
* struct

but are inline for control elements like:
* if
* for
* while